example-mlem-get-started/code/README.md

```diff
@@ -0,0 +1,37 @@
+# MLEM Get Started
+
+This is an auto-generated repository for use in MLEM
+[Get Started](https://mlem.ai/doc/get-started). It is a step-by-step quick
+introduction into basic MLEM concepts.
+
+🐛 Please report any issues found in this project here -
+[example-repos-dev](https://github.com/iterative/example-repos-dev).
+
+## Installation
+
+Python 3.6+ is required to run code from this repo.
+
+```console
+$ git clone https://github.com/iterative/example-mlem-get-started
+$ cd example-mlem-get-started
+```
+
+Now let's install the requirements. But before we do that, we **strongly**
+recommend creating a virtual environment with a tool such as
+[virtualenv](https://virtualenv.pypa.io/en/stable/):
+
+```console
+$ virtualenv -p python3 .env
+$ source .env/bin/activate
+$ pip install -r src/requirements.txt
+```
+
+## Existing stages
+
+This project with the help of the Git tags reflects the sequence of actions that
+are run in the MLEM [get started](https://mlem.ai/doc/get-started) guide. Feel
+free to checkout one of them and play with the MLEM commands having the
+playground ready.
+
+[comment]: <> (TODO)
```

Comment:
resolve this, not sure this is needed at all in this case - it's simpler than the DVC's project